charts/bitcoind/values.yaml

```diff
@@ -139,3 +139,6 @@ bitcoindCustomConfig:
   bind: 0.0.0.0
   rpcbind: 0.0.0.0
   rpcallowip: 0.0.0.0/0
+
+descriptorImport:
+  enabled: false
```

Comment:
remove in favour of `descriptor.{secretName,secretKey}`

Reply:
done, the secretName,secretKey are given as empty by default